.gitignore

```diff
@@ -1,3 +1,9 @@
+/node_modules
+/build
+.env
+yarn-debug.log*
+yarn-error.log*
```

Comment:
These entries are irrelevant for translated-content.

```suggestion
```

Reply:
Okay, let me make an adjustment.